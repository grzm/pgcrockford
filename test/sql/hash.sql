-- reference
SELECT hashint4(55);
SELECT hashoid(55);

SELECT crockford.hashcrockford2('1Q'::crockford.crockford2);
SELECT crockford.hashcrockford4('1Q'::crockford.crockford4);
SELECT crockford.hashcrockford8('1Q'::crockford.crockford8);
