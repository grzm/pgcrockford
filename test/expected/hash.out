-- reference
SELECT hashint4(55);
  hashint4  
------------
 -305105437
(1 row)

SELECT hashoid(55);
  hashoid   
------------
 -305105437
(1 row)

SELECT crockford.hashcrockford2('1Q'::crockford.crockford2);
 hashcrockford2 
----------------
     -305105437
(1 row)

SELECT crockford.hashcrockford4('1Q'::crockford.crockford4);
 hashcrockford4 
----------------
     -305105437
(1 row)

SELECT crockford.hashcrockford8('1Q'::crockford.crockford8);
 hashcrockford8 
----------------
     -305105437
(1 row)

